src/testing/ui-test-playwright/playeruserflow.spec.js

```diff
@@ -10,30 +10,40 @@
 
 const { test, expect } = require('@playwright/test');
 
+const baseUrl = process.env.TEST_BASEURL; // Needs to include protocol, e.g. https://
+
 // If this test file is being used for actual UI testing, set these values both to 1sec
 const minTaskWaitSeconds = process.env.TEST_MIN_TASK_WAIT_SECONDS || 3;
```

Comment:
@msimecek I was thinking whether to make the defaults to conform more for actual testing than for load gen? what do you think?

Reply:
The idea is that UI test should be quick, while load test should be more realistic, right? Either way works, you'll have to configure it at least once anyway.

Reply:
Thinking about it a bit more - you're probably right. For UI testing there's no point in touching this config, while for load test it could be potentially tweaked more (therefore the defaul would be better for UI testing).

Reply:
ok, will adjust it

Reply:
ok done. Also found a way to count available items for picking